### ISSUE TITLE
Filter media files the same as MPDN

### ISSUE THREAD

**Issue by Ryrynz** [Garteal/MPDN_Extensions#16]
.swf flash files are visible to the playlist open window and are also added automatically when play next file in folder is active even though MPDN doesn't support this file type.

**Comment by Garteal** [comment]
@zachsaw any chance I could get the complete file supported list?

**Comment by Ryrynz** [comment]
I would like to see MPDN expose it's supported files so that any additions to it simply carry over. I've asked Zach about supporting playlist files natively also.

**Comment by Garteal** [comment]
Good idea. Didn't really spend too much time on it as I simply took what was already there and only moved it around in the code. Guess this'll happen once we hear from Zachs.

**Comment by zachsaw** [comment]
MPDN's "supported" file extensions list isn't very complete. I might add something in PlayerControl to return the list of extensions, and also register additional file types.
